rinja/src/filters/escape.rs

```diff
@@ -486,6 +476,12 @@ where
 {
 }
 
+crate::impl_for_ref! {
```

Comment:
Do we have tests already for ensuring it's working for all references?

Reply:
Good call! No, there weren't any tests. Added.